build.gradle

```diff
@@ -17,9 +17,9 @@ repositories {
 }
 
 
-sourceCompatibility = 11
+sourceCompatibility = 17
```

Comment:
Do we really need to switch to java 17?

Reply:
Actually, it breaks the build as java 11 is used as GH action dependency.

Reply:
Good point with the eol for java 11. Please update the GH workflows as well to use java 17 then.

Reply:
I see, you've just reverted, so let's leave it with 11, and fix that in future PR.